src/Config.ts

```diff
@@ -39,15 +41,20 @@ export default function ClientConfig(opts = {}) {
         // For ethers.js provider params, see https://docs.ethers.io/ethers.js/v5-beta/api-providers.html#provider
         mainnet: null, // Default to ethers.js default provider settings
         sidechain: {
+            // @ts-expect-error
             url: null, // TODO: add our default public service sidechain node, also find good PoA params below
             // timeout:
             // pollingInterval:
         },
+        // @ts-expect-error
         dataUnion: null, // Give a "default target" of all data union endpoint operations (no need to pass argument every time)
         tokenAddress: '0x0Cf0Ee63788A0849fE5297F3407f701E122cC023',
         minimumWithdrawTokenWei: '1000000', // Threshold value set in AMB configs, smallest token amount to pass over the bridge
+        // @ts-expect-error
```

Comment:
Yeah these should probably be undefined or set to some default value or fix the type to be `| null` for now, IMO better than `// @ts-expect-error`